OpenOversight/app/templates/officer.html

```diff
@@ -13,16 +13,33 @@ <h1>Officer Detail: <b>{% include 'partials/officer_name.html' %}</b></h1>
 
   <div class="row">
     <div class="col-sm-6">
+      {# Featured tag comes first #}
       {% for path in paths %}
-      {% if current_user.is_administrator
-        or (current_user.is_area_coordinator and current_user.ac_department_id == officer.department_id) %}
-      <a href="{{ url_for('main.display_tag', tag_id=faces[loop.index - 1].id) }}">
-      {% endif %}
-      <img src="{{ path }}" alt="Submission">
-      {% if current_user.is_administrator
-        or (current_user.is_area_coordinator and current_user.ac_department_id == officer.department_id) %}
-      </a>
-      {% endif %}
+        {% if faces[loop.index - 1].featured == True %}
+          {% if current_user.is_administrator
+            or (current_user.is_area_coordinator and current_user.ac_department_id == officer.department_id) %}
+          <a href="{{ url_for('main.display_tag', tag_id=faces[loop.index - 1].id) }}">
+          {% endif %}
+          <img src="{{ path }}" alt="Submission">
+          {% if current_user.is_administrator
+            or (current_user.is_area_coordinator and current_user.ac_department_id == officer.department_id) %}
+          </a>
+          {% endif %}
+        {% endif %}
+      {% endfor %}
+      {# All non-featured tags #}
+      {% for path in paths %}
+        {% if faces[loop.index - 1].featured == False %}
+          {% if current_user.is_administrator
+            or (current_user.is_area_coordinator and current_user.ac_department_id == officer.department_id) %}
+          <a href="{{ url_for('main.display_tag', tag_id=faces[loop.index - 1].id) }}">
+          {% endif %}
+          <img src="{{ path }}" alt="Submission">
+          {% if current_user.is_administrator
+            or (current_user.is_area_coordinator and current_user.ac_department_id == officer.department_id) %}
+          </a>
+          {% endif %}
+        {% endif %}
```

Comment:
I am not a fan of putting yet more logic into templates. Why not just add a ORDER BY `featured` to where we retrieve the Faces? Then, I don't think we need any of the chances here

Reply:
yup that's definitely a better way to go